core/src/main/java/org/elasticsearch/action/bulk/BulkProcessor.java

```diff
@@ -79,6 +78,7 @@
         private int bulkActions = 1000;
         private ByteSizeValue bulkSize = new ByteSizeValue(5, ByteSizeUnit.MB);
         private TimeValue flushInterval = null;
+        private BackoffPolicy backoffPolicy = BackoffPolicy.noBackoff();
```

Comment:
I know I'm late to the party, but the point of #14620 was to have a default that does back off and retry. Why was a no back off chosen as a default?